controllers/argocd/sso/dex/constants.go

```diff
@@ -0,0 +1,5 @@
+package dex
+
+const (
+	ArgoCDDexControllerComponent = "dex-server"
+)
```

Comment:
does it make sense to combine `constants` into `util` so to have fewer file to manage, as the above constant is only used in util.